client/pages/index.js

```diff
@@ -24,21 +44,38 @@ function Index() {
     
     get();
   }, [position]) 
+
   return (
-    <>
-      <main>
+    <div style={{ display: "flex", height: "100vh" }}>
+      <div className="sidebar" style={{ width: "20%", backgroundColor: "Black", padding: "20px" }}>
+        {/* Sidebar content goes here */}
+        <h1 style ={{padding: "10px 20px", textAlign: "center", fontWeight: "bold", color: "white", fontSize: "1.5em"}}>Food Truck Finder</h1>
+        <input
+          type="text"
+          value={searchTerm}
+          onChange={(e) => setSearchTerm(e.target.value)}
+          placeholder="Search Food Trucks"
+          style={{ width: "100%", marginBottom: "10px", padding: "10px 20px"}}
+        />
+        <ul style={{textAlign: "left", color: "white"}}>
+          {searchFoodTrucks.map((foodTruck) => (
+            <li key={foodTruck.id}>{foodTruck.name}</li>
+          ))}
+        </ul>
+      </div>
```

Comment:
Think about building this as its own react component. Eventually index.js will be too crowded, its already a bit cluttered
Same goes for the map, that should be a component